packages/docs/src/routes/tutorial/hooks/use-visible-task/solution/app.tsx

```diff
@@ -15,11 +15,9 @@ export const Clock = component$(() => {
     second: 0,
   });
 
-  useVisibleTask$(({ track }) => {
-    track(store);
-    updateClock(store);
-    const tmrId = setTimeout(() => updateClock(store), 1000);
-    return () => clearTimeout(tmrId);
+  useVisibleTask$(() => {
+    const intervalId = setInterval(() => updateClock(store), 1000);
+    return () => clearTimeout(intervalId);
```

Comment:
it's working but i'd use `clearInterval` instead 😄

Reply:
Oops, sorry about that!
I guess I was so focused on following the PR guidelines that I overlooked the actual code haha 🤦‍♂️